Assets/Sprites/topdowntanks/PNG/Environment/treeLarge.png.meta

```diff
@@ -40,7 +40,7 @@ TextureImporter:
   alignment: 0
   spritePivot: {x: 0.5, y: 0.5}
   spriteBorder: {x: 0, y: 0, z: 0, w: 0}
-  spritePixelsToUnits: 100
+  spritePixelsToUnits: 107
```

Comment:
Would recommend also using 128. Consistent and always fits nicely on cubes below and tiles